### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by fourbytes**
- Bump `redis` to `0.21`
- Bump `env_logger` to `0.9`

### CODE REVIEW

**Comment by epwalsh** [comment]
Hey @fourbytes, can you update your branch when you get a chance? I don't see the "Update Branch" button, did you not enable edits for maintainers?

**Comment by fourbytes** [comment]
> Hey @fourbytes, can you update your branch when you get a chance? I don't see the "Update Branch" button, did you not enable edits for maintainers?

Ah it appears that option is only available for user-owned forks, and this one is under an organization. I've rebased my branch with upstream.

**Review by epwalsh — APPROVED** [review]
LGTM as long as checks pass!

**Pull request merged**
